curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/657d491c439d642176fc2fc8.md

```diff
@@ -0,0 +1,387 @@
+---
+id: 657d491c439d642176fc2fc8
+title: Step 54
+challengeType: 0
+dashedName: step-54
+---
+
+# --description--
+
+Next, you will need to update the total score text content with the current value of `totalScore`.
+
+Below that, you will need to add to the `scoreHistory` HTML content with the following template literal: `<li>${achieved} : ${selectedValue}</li>`.
+
+# --hints--
+
+You should use the `textContent` property to update the `totalScoreText` variable to display the `totalScore` variable.
+
+```js
+assert.match(code, /totalScoreText\s*\.\s*textContent\s*=\s*totalScore\s*;?/);
+```
+
+You should use the `innerHTML` property to set the `scoreHistory` to display `<li>${achieved} : ${selectedValue}</li>`. 
+
+```js
+assert.match(code, /scoreHistory\s*\.\s*innerHTML\s*[\+\=]+\s*`<li>\s*\$\{\s*achieved\s*\}\s*:\s*\$\{\s*selectedValue\s*\}\s*<\/li>`\s*;?/);
+```
```

Comment:
Edited the hint text for clarity, and updated the regex to look for `+=`:

````suggestion
You should use the `+=` operator to append `<li>${achieved} : ${selectedValue}</li>` to the `innerHTML` property of `scoreHistory`.

```js
assert.match(code, /scoreHistory\s*\.\s*innerHTML\s*\+\s*\=\s*`<li>\s*\$\{\s*achieved\s*\}\s*:\s*\$\{\s*selectedValue\s*\}\s*<\/li>`\s*;?/);
```
````